### PR TITLE
Glossary + FAQ v2

### PR DESCRIPTION
**Pull request by chasulin**
[Trello item](https://trello.com/c/hLjtHbMg/20-glossary-v2)

Applies new format to the glossary + FAQ (removes accordions, adds a 'scroll to top' button on mobile to solve for the heavy amount of content).

Todo:
- [x] Apply this format to FAQ
- [x] Add tracking

### CODE REVIEW

File: src/common/hooks/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import useScrollToTopButton from './useScrollToTopButton';
```
Comment (chasulin) [inline src/common/hooks/index.ts:1]: a centralized hooks file can be useful in the process of unifying our styles and components. i see there is also a useEmbed hook that ill move into this file.

**Review by pnavarrc — COMMENTED** [review]
First pass, looks good, I proposed some changes to how the scroll to top button and behaviours are structured

File: src/common/hooks/useBreakpoint.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { useMediaQuery } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
+
+export default function useBreakpoint(breakpointWidth: number): boolean {
+  const theme = useTheme();
+  const isBelowBreakpoint = useMediaQuery(
+    theme.breakpoints.down(breakpointWidth),
+  );
+  return isBelowBreakpoint;
+}
```
Comment (chasulin) [inline src/common/hooks/useBreakpoint.ts:12]: gonna sprinkle this around + replace the many instances where ive written/repeated the logic here

File: src/components/SharedComponents/ScrollToTopButton.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
+
+const ScrollToTopButton = (props: {
+  showButton: boolean;
+  analyticsCategory: EventCategory;
+}) => {
+  const { showButton, analyticsCategory } = props;
+
+  const onClickHandler = () => {
+    trackEvent(analyticsCategory, EventAction.CLICK, 'Scroll to top button');
```
Comment (pnavarrc) [inline src/components/SharedComponents/ScrollToTopButton.tsx:12]: maybe just 'Scroll to Top', so the label don't use that much space on the GA reports

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Pull request merged**